File: Net/01/Tasks.Domain/Commands/ChangeTaskTitle.cs
using Cqrs;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Tasks.Commands
{
    public class ChangeTaskTitle : ICommand
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
    }
}

File: Java/01.ActiveMq/context-tasks/src/main/java/org/cqrs101/tasks/Commands/ChangeTaskTitle.java
package org.cqrs101.tasks.Commands;

import java.util.UUID;
import org.cqrs.Command;

public class ChangeTaskTitle implements Command {

    private UUID id;
    private String title;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}
